numpy/random/tests/test_generator_mt19937.py

```diff
@@ -898,9 +898,9 @@ def test_gumbel_0(self):
     def test_hypergeometric(self):
         random.bit_generator.seed(self.seed)
         actual = random.hypergeometric(10.1, 5.5, 14, size=(3, 2))
-        desired = np.array([[10, 10],
-                            [10, 10],
-                            [9, 9]])
+        desired = np.array([[9, 9],
+                            [10, 9],
+                            [9, 10]])
```

Comment:
Why does the result change?

Reply:
There was a bug in the old HRUA code that made it less efficient than it should have been.  HRUA is a rejection method, and with the fixed formula, the new code will generally have fewer rejections than the old code.  That means that even with the same bit generator and the same seed, the stream of variates out of the hypergeometric distribution will be different.  I didn't touch the legacy version (other than to move it to legacy-distributions.c), so the variates from the legacy hypergeometric distribution in RandomState haven't changed.